config/params.yaml

```diff
@@ -8,6 +8,15 @@ angle_error_adjustment: 0.0 #radians TUNING
 max_lidar_range: 29 #meters - should be slightly less than real range, or accurate range 
 min_lidar_range: 0.1 #meters - should be slight larger than real, or accurate range
 
+min_circle_pts: 6 # minimum number of points necessary for radius checking
+marker_radius: 0.127 #meters
+large_buoy_radius: 0.184 #meters
+small_buoy_radius: 0.1015 #meters
```

Comment:
I don't think we are going to use prop_mapper for small buoys, it might be better for now to just use "buoy_radius" instead of small and large, and avoid having an extra param in there that's not currently being used. What do you think?